snapshots/oecd/2025-03-27/govt_glance_size_public_procurement.py

```diff
@@ -0,0 +1,24 @@
+"""Script to create a snapshot of dataset."""
+
+from pathlib import Path
+
+import click
+
+from etl.snapshot import Snapshot
+
+# Version for current snapshot dataset.
+SNAPSHOT_VERSION = Path(__file__).parent.name
+
+
+@click.command()
+@click.option("--upload/--skip-upload", default=True, type=bool, help="Upload dataset to Snapshot")
+def run(upload: bool) -> None:
```

Comment:
If you are planning on updating the three files always at the same time, you could have created just one .py file, to fetch the three (having the same version). But you can keep it this way.